src/common.c

```diff
@@ -62,10 +63,14 @@ int fi_wait_cond(pthread_cond_t *cond, pthread_mutex_t *mut, int timeout)
 	ts.tv_sec += timeout / 1000;
 	ts.tv_nsec += (timeout % 1000) * 1000000;
 	return pthread_cond_timedwait(cond, mut, &ts);
+#else /* _WIN32 */
+	return !SleepConditionVariableCS(cond, mut, (DWORD)timeout);
+#endif /* _WIN32 */
```

Comment:
easier to define pthread_cond_timedwait in windows along with the other pthread values.